lib/linguist/heuristics.yml

```diff
@@ -28,6 +28,29 @@
 #
 ---
 disambiguations:
+- extensions: ['.1', '.1in', '.1m', '.1x', '.2', '.3', '.3in', '.3m', '.3qt', '.3x', '.4', '.5', '.6', '.7', '.8', '.9', '.man', '.mdoc']
```

Comment:
@Alhadis would it work also for manpages if their file name ends with `.8.in`?

Reply:
No, because handling `.in` and other unrecognised extensions will be the responsibility of a dedicated man page strategy. 👍

Reply:
So support for `.8.in` manpages would become later, right?

Reply:
Yes. 👍